.drone.yml

```
@@ -80,372 +77,340 @@ steps:
     name: check-deprecated-apis-nginx
     environment:
       KUBERNETES_MANIFESTS: nginx.yml
-
 ---
-name: e2e-kubernetes-1.25
 kind: pipeline
 type: docker
-
-depends_on:
-  - policeman
-
-node:
-  runner: internal
-
+name: E2E Tests on kind clusters version 1.26.6
 platform:
   os: linux
   arch: amd64
-
+depends_on:
+  - policeman
+clone:
+  depth: 1
 trigger:
   ref:
     include:
-      - refs/heads/master
       - refs/heads/main
       - refs/tags/**
-
+volumes:
+  - name: dockersock
+    host:
+      path: /var/run/docker.sock
+  - name: tmp-kind
+    temp: {}
 steps:
-  - name: init
-    image: quay.io/sighup/e2e-testing-drone-plugin:v1.25.3
+  - name: Generate Kind Config file
+    image: alpine:latest
     pull: always
-    volumes:
-      - name: shared
-        path: /shared
-    depends_on: [clone]
-    settings:
-      action: custom-cluster-125
-      pipeline_id: cluster-125
-      local_kind_config_path: katalog/tests/kind/config.yml
-      cluster_version: "1.25.3"
-      instance_path: /shared
-      aws_default_region:
-        from_secret: aws_region
-      aws_access_key_id:
-        from_secret: aws_access_key_id
-      aws_secret_access_key:
-        from_secret: aws_secret_access_key
-      terraform_tf_states_bucket_name:
-        from_secret: terraform_tf_states_bucket_name
-      vsphere_server:
-        from_secret: vsphere_server
-      vsphere_password:
-        from_secret: vsphere_password
-      vsphere_user:
-        from_secret: vsphere_user
-      dockerhub_username:
-        from_secret: dockerhub_username
-      dockerhub_password:
-        from_secret: dockerhub_password
-
-  - name: e2e
-    # KUBECTL 1.25.3 - KUSTOMIZE 3.5.3 - HELM 3.1.1 - YQ 4.21.1 - ISTIOCTL 1.9.4 - FURYCTL 0.9.0 - BATS 1.1.0
-    image: quay.io/sighup/e2e-testing:1.1.0_0.9.0_3.1.1_1.9.4_1.25.3_3.5.3_4.21.1
+    commands:
+      - sh ./katalog/tests/kind/generate-template.sh 1.26.6
+    environment:
+      KUBERNETES_VERSION: 1.26.6
+  - name: Create Kind Cluster
```

I'd suggest leaving an empty line between steps for better readability
```suggestion
      KUBERNETES_VERSION: 1.26.6

  - name: Create Kind Cluster
```